practice/src/index.html

```diff
@@ -7,10 +7,58 @@
         <title>Todo List app - TypeScript</title>
         <link rel="stylesheet" href="./styles/index.css">
         <link rel="shortcut icon" href="./assets/favicon.8602d591.ico" type="image/x-icon">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" />
     </head>
     <body>
-        
+        <main class="app">
+            <header class="app-heading">
+                <h1 class="heading-title">Todo list with TypeScript</h1>
+                <p class="heading-description">
+                    Build from <a href="https://www.typescriptlang.org/" class="underline-text" target="_blank">TypeScript</a>
+                    and <a href="https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller" class="underline-text" target="_blank">MVC model</a>
+                </p>
+            </header>
+            <div class="app-todo-content">
+                <div class="content-input">
+                    <i class="fa fa-check check-all"></i>
```

Comment:
should close tag if it isn't have content after
example `<i class="fa fa-check check-all" />`
the same another case